content/kubermatic/v2.18/architecture/known_issues/_index.en.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Known Issues"
+date = 2022-07-20T12:22:15+02:00
+weight = 1
+
++++
+
+## Overview
+
+This page documents the list of known issues description and possible work arounds/solutions.
+
+### 1. OIDC user authentication issue
+
+**Problem** 
+
+OIDC user is denied to access the user cluster in the KKP with K8s version of 1.20 and below. Refer the github issue [Bug: OIDC authentication...](https://github.com/kubermatic/kubermatic/issues/9908) for detailed problem description.
```

Comment:
Please also mention the log message that would appear in the apiserver logs in case of triggering this issue, e.g.:
```
oidc authenticator: initializing plugin: Get "https://<kkp-url>/dex/.well-known/openid-configuration": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
```